06-Modelling-Rigid-Bodies/03-Preparing-a-General-Body.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>further reading link looks broken.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Indeed. I moved the link to "parallel axis theorem", but failed to delete the "further reading" text. Fixed.</p>